packages/buttons/src/views/Anchor.js

```diff
@@ -34,30 +36,51 @@ const StyledAnchor = styled.a.attrs({
   ${props => retrieveTheme(COMPONENT_ID, props)};
 `;
 
+const StyledNewWindowIcon = styled(NewWindowIcon)`
```

Comment:
Do we need to add any aria attributes here to make sure this doesn't get screen reader confusion?

Reply:
> Do we need to add any aria attributes here to make sure this doesn't get screen reader confusion?

I couldn't find any consensus on this in the WAI-ARIA docs and other resources. Personally, I feel that since the screen-reader already says the url location and whether it will open in a new tab/window through the `target` attribute that it could be ok.

> Somehow that seems to be getting stripped in the (SVGO?) build process – and it means that this icon is focusable in IE (which we don't want)

I've updated our styleguidist and webpack builds to directly reference the `@zendeskgarden/svg-icons/.svgo.yml` file. Is that ok to do?

Reply:
I wouldn't recommend referencing that config directly. Use only what you need. For example, we do want `svg-icons` to be pretty printed for ease of consumer inspection. But we don't want that in our build.

Reply:
Your updates lost the `viewBox` – that's why the SVG isn't sizing properly.